state/settings.go

```diff
@@ -9,6 +9,7 @@ import (
 	"sort"
 
 	"github.com/juju/errors"
+	statetxn "github.com/juju/txn"
```

Comment:
this import alias looks a bit wrong

Reply:
In that particular package it is usually referenced as statetxn (many occurrences, typically in test files) and jujutxn. I prefer the latter but picked the former for consistency; happy to change to jujutxn.

Reply:
https://github.com/juju/juju/pull/11204/commits/5517d74c8c0525437ee86833c30c4c326603f217 replaces all statetxn aliases with jujutxn.